Code/Framework/AzToolsFramework/AzToolsFramework/Input/QtEventToAzInputManager.cpp

```diff
@@ -381,25 +398,73 @@ namespace AzToolsFramework
         return QPoint{ denormalizedX, denormalizedY };
     }
 
+    void wrapCursorX(const QRect& rect, QPoint& point) {
+        if (rect.left() < point.x()) 
+        {
+            point.setX(rect.right() - 1);
+        }
+        else if (rect.right() > point.x()) 
+        {
+            point.setX(rect.left() + 1);
+        }
+    }
+
+    void wrapCursorY(const QRect& rect, QPoint& point) {
+        if (rect.top() < point.y()) 
+        {
+            point.setY(rect.bottom() - 1);
+        }
+        else if (rect.bottom() > point.y()) 
+        {
+            point.setY(rect.top() + 1);
+        }
+    }
+
+
     void QtEventToAzInputMapper::HandleMouseMoveEvent(const QPoint& globalCursorPosition)
     {
         const QPoint cursorDelta = globalCursorPosition - m_previousGlobalCursorPosition;
+        QScreen* screen = m_sourceWidget->screen();
```

Comment:
nit: This could be `const`

Reply:
you can't do that QCursor::setPos first parameter is not const.

Reply:
You're quite right, I do apologize 😝 Thanks!